### PR TITLE
Adding place-self property

### PR DESCRIPTION
**Pull request by davidbernegger**
As requested #253 #292 and  Microsoft/vscode-css-languageservice#117

### CODE REVIEW

**Comment by connorshea** [comment]
For reviewers:

- https://drafts.csswg.org/css-align-3/#place-self-property
- https://developer.mozilla.org/en-US/docs/Web/CSS/place-self

File: css/properties.json [hunk 1]
```diff
@@ -6806,6 +6806,28 @@
...
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Box Alignment"
+    ],
+    "initial": [
+      "align-self",
+      "justify-self"
+    ],
+    "appliesto": "allElements",
```
Comment (connorshea) [inline css/properties.json]: The spec says that it applies to "block-level boxes, absolutely-positioned boxes, and grid items", not all elements.

https://www.w3.org/TR/css-align-3/#place-self-property

**Comment by octref** [comment]
Just subscribe myself here, also for https://github.com/Microsoft/vscode/issues/64835.

File: css/properties.json (hunk 1 above)
Comment (ExE-Boss) [reply css/properties.json]:
```suggestion
    "appliesto": "blockLevelBoxesAndAbsolutelyPositionedBoxesAndGridItems",
```

**Comment by ExE-Boss** [comment]
You can just accept my suggestion from the&nbsp;Github&nbsp;UI.

**Comment by davidbernegger** [comment]
Just updated it with your suggestion @ExE-Boss

**Comment by chrisdavidmills** [comment]
This looks good; thanks @davidbernegger !

**Comment by davidbernegger** [comment]
@chrisdavidmills:  @connorshea's pull request #253 could be closed.

**Pull request merged**